### ISSUE TITLE
Allow town or nation access in discord

### ISSUE THREAD

**Issue by R3dan** [EarthMC/Issue-Tracker#1706]
### Is your feature request related to a problem? Please describe.

I would like to be able to use the commands given to me in a town (either as mayor or ranked) in discord. Allow someone to change the town board etc on discord or invite players similar to discord server invite link. Obviously you would have to link your Minecraft and discord account but you could do that with the bot.

### Describe the solution you'd like

Basically be able to use towny commands in discord

**Comment by 3meraldK** [comment]
No pal, discord is entirely different thing than a minecraft server. Voting negatively

**Comment by Veyronity** [comment]
No, that justt encourages alting

**Comment by Veyronity** [comment]
Just get online and do your commands there, lazy
